tests/test-import.js

```diff
@@ -18,6 +18,24 @@ describe('layout: importer', function () {
   const dirBig = __dirname + '/test-data/dir-big'
   const dirNested = __dirname + '/test-data/dir-nested'
 
+  // check to see if missing empty dirs need to be created
+  fs.exists(__dirname+'/test-data/dir-nested/dir-another', function(exists){
+    if(!exists){
+      fs.mkdir(__dirname+'/test-data/dir-nested/dir-another', function(){
+        fs.exists(__dirname+'/test-data/dir-nested/level-1/level-2', function(exists){
```

Comment:
this is not dependent of creating `dir-another`. You can just leave leave lines 32-35 to do it